_includes/header.html

```diff
@@ -29,4 +29,4 @@
             </form>
```

Comment:
Can you update the donate page here to point to your new donate page?

Reply:
Done.